install/installer/pkg/components/proxy/service.go

```diff
@@ -15,6 +16,19 @@ import (
 )
 
 func service(ctx *common.RenderContext) ([]runtime.Object, error) {
+	serviceType := corev1.ServiceTypeLoadBalancer
+	_ = ctx.WithExperimental(func(cfg *experimental.Config) error {
+		if cfg.Common != nil && cfg.Common.ServiceConfig != nil {
+			st, ok := cfg.Common.ServiceConfig["proxy"]
+			if ok {
+				if strings.ToLower(st.ServiceType) == "clusterip" {
```

Comment:
I see this conditional doesn't have an else clause; if `ServiceType` is misspelled (eg `cluterip`) will this this code default to `ServiceTypeLoadBalancer`? Is there validation occurring in this pipeline?

Reply:
This is a great point Adrien! I agree that we can add a validation process. As of this implementation, it just defaults to `LoadBalancer` as set in [this line](https://github.com/gitpod-io/gitpod/blob/main/install/installer/pkg/config/v1/experimental/validation.go).

About the validation, it is a great tip! We can leverage the existing [`validate` cmd](https://github.com/gitpod-io/gitpod/blob/main/install/installer/pkg/config/v1/experimental/validation.go)! I will do a follow-up PR for this!